static/app/views/performance/onboarding.tsx

```diff
@@ -733,14 +735,35 @@ export function Onboarding({organization, project}: OnboardingProps) {
             </div>
             <GuidedSteps.ButtonWrapper>
               <GuidedSteps.BackButton size="md" />
-              <SampleButton
-                triggerText={t('Take me to an example')}
-                loadingMessage={t('Processing sample trace...')}
-                errorMessage={t('Failed to create sample trace')}
-                organization={organization}
-                project={project}
-                api={api}
-              />
+              {received ? (
+                <Button
+                  priority="primary"
+                  onClick={() => {
+                    navigate(
+                      {
+                        pathname: location.pathname,
+                        query: {
+                          ...location.query,
+                          guidedStep: undefined,
+                        },
+                      },
+                      {replace: true}
+                    );
+                    window.location.reload();
+                  }}
+                >
+                  {t('Take me to traces')}
```

Comment:
If we want to be consistent with the Issues onboarding, we probably want to capitalize traces here
```suggestion
                  {t('Take me to Traces')}
```